lstchain/io/tests/test_eventselection.py

```diff
@@ -38,33 +38,69 @@ def test_event_selection():
     )
 
 
-def test_dl3_fixed_cuts():
-    temp_cuts = DL3FixedCuts()
+def test_dl3_global_cuts():
+    temp_cuts = DL3Cuts()
 
-    temp_cuts.fixed_gh_cut = 0.7
-    temp_cuts.fixed_theta_cut = 0.2
+    temp_cuts.global_gh_cut = 0.7
+    temp_cuts.global_theta_cut = 0.2
     temp_cuts.allowed_tels = [1, 2]
+    temp_cuts.gh_max_efficiency = 0.8
+    temp_cuts.theta_containment = 68
 
     temp_data = QTable({
-        "gh_score": u.Quantity(np.arange(0.1, 1.1, 0.1)),
-        "theta": u.Quantity(np.arange(0., 1., 0.1), unit=u.deg),
-        "tel_id": u.Quantity([1, 1, 2, 2, 1, 2, 1, 3, 4, 5])
+        "gh_score": u.Quantity(np.tile(np.arange(0.35, 0.85, 0.05), 3)),
+        "reco_energy": u.Quantity(np.logspace(-2.301, 1.699, 30), unit=u.TeV),
```

Comment:
Why -2.301  to  1.699 ? (instead of -2.3, 1.7,  I mean)

Reply:
I guess copy paste from the EventDisplay definitions.

Reply:
Nothing particular, just wanted the values to be 'slightly more' close to a nice integral value. it should work fine with either

Reply:
That should also be 31 bin edges to get 30 bins in energy. And if you were aiming for "five bins per decade", that is only 4 decades so you'd need only 20 bins.

Better use `np.geomspace` to get human readable values here:

`np.geomspace(5 * u.GeV, 50 * u.TeV, 31)`

Reply:
> Nothing particular, just wanted the values to be 'slightly more' close to a nice integral value. it should work fine with either

I always fear such numbers that in any plot will "look like" exactly -2.3 and 1.7,  but are not. Then, you will anyway not have integer energy numbers (in GeV - is that what you meant?) for all bin edges, given the log scale... Also, if I see such non-round numbers in the code I immediately think there is a reason behind them.  So it is somewhat confusing.